test/quantization/test_quantized_tensor.py

```diff
@@ -213,6 +181,38 @@ def _test_qtensor(self, device):
                              "quantization_scheme=torch.per_tensor_affine, " +
                              "scale=1.0, zero_point=2)")
 
+    def test_cuda_quantized_tensor(self):
```

Comment:
is this code copied from L149:L180, or are there any changes as well?

Reply:
I'm not 100% sure what happened, but I tried to rebase it, and so in this PR only the changes since 58245 are showing up. Will fix and try to create a new PR